src/Parsers/ExpressionElementParsers.cpp

```diff
@@ -105,62 +105,70 @@ bool ParserSubquery::parseImpl(Pos & pos, ASTPtr & node, Expected & expected)
 {
     ParserSelectWithUnionQuery select;
     ParserExplainQuery explain;
-
-    if (pos->type != TokenType::OpeningRoundBracket)
-        return false;
-    ++pos;
-
     ASTPtr result_node = nullptr;
+    ParserKeyword s_kql("KQL");
 
-    if (ASTPtr select_node; select.parse(pos, select_node, expected))
+    if (s_kql.ignore(pos, expected))
     {
-        result_node = std::move(select_node);
+        if (!ParserKQLTaleFunction().parse(pos, result_node, expected))
```

Comment:
I believe it's a typo - should be `ParserKQLTableFunction`

Reply:
will fix